pkg/log/log.go

```diff
@@ -55,9 +55,9 @@ func Init(withName string, out io.Writer) {
 		// be propagated through the whole operator, generating
 		// uniform and structured logs.
 		if out == nil {
-			logf.SetLogger(zap.Logger())
+			logf.SetLogger(zap.Logger(false))
```

Comment:
the doc says that this `Logger(..)` func is now deprecated and suggest to use `New()` instead